src/main/java/com/unstoppabledomains/resolution/Namehash.java

```diff
@@ -15,15 +15,16 @@
 import com.unstoppabledomains.resolution.artifacts.Numeric;
 
 
-public class NameHash {
+public class Namehash {
+    private Namehash() {}
```

Comment:
This constructor is unneeded since we're using only `static` metods.

Reply:
UPD: I can see that it's `private`, I see your purpose. Ignore the previous message.